gensim/models/hdpmodel.py

```diff
@@ -507,9 +508,13 @@ def save_options(self):
             fout.write('eta: %s\n' % str(self.m_eta))
             fout.write('gamma: %s\n' % str(self.m_gamma))
 
-    def hdp_to_lda(self):
+    def hdp_to_lda(self, model=False):
```

Comment:
It would be better to have a new method called "suggested_lda_model" that returns the model with reshaped alpha